code/src/sixsq/nuvla/server/resources/spec/notification.cljc

```diff
@@ -0,0 +1,91 @@
+(ns sixsq.nuvla.server.resources.spec.notification
+  (:require
+    [clojure.spec.alpha :as s]
+    [sixsq.nuvla.server.resources.spec.common :as common]
+    [sixsq.nuvla.server.resources.spec.core :as core]
+    [sixsq.nuvla.server.util.spec :as su]
+    [spec-tools.core :as st]))
+
+
+(s/def ::message
+  (-> (st/spec ::core/nonblank-string)
+      (assoc :name "message"
+             :json-schema/type "string"
```

Comment:
Should be inherited from nonblank-string.  Can be removed.

Reply:
Addressed in 6c6f792